src/test/java/vaildator/CommandValidatorTest.java

```diff
@@ -151,15 +151,4 @@ public void isValidAddCommand_invalidCommand_expectFalse() {
         assertFalse(CommandValidator.isValidAddCommand(test));
     }
 
-    @Test
-    public void isValidAddEntries_validEntries_expectedTrue() {
-        String test = "add pizza, abc, 340, egg nut dairy gluten, dinner, www.food.com";
-        assertTrue(CommandValidator.isValidAddEntries(test));
-    }
```

Comment:
also, why did you take out these tests?

Reply:
I found out that the special characters were handled at the ```isName()``` method, which covers more than what I did. Hence, I removed my part so that the code could be cleaner.